x-pack/test/cloud_security_posture_functional/page_objects/add_cis_integration_form_page.ts

```diff
@@ -178,6 +179,8 @@ export function AddCisIntegrationFormPageProvider({
   const clickLaunchAndGetCurrentUrl = async (buttonId: string) => {
     const button = await testSubjects.find(buttonId);
     await button.click();
+    // Wait a bit to allow the new tab to load the URL
+    await sleep(3000);
```

Comment:
++ I guess for those cases it's ok to have a delay as opening a new window is not a synchronous function